src/btree/col_srch.c

```diff
@@ -328,6 +338,13 @@ __wt_col_search(
             cbt->compare = 0;
         else if (recno < cbt->recno)
             cbt->compare = 1;
+        /*
+         * Special case for FLCS: the searched-for record number is greater then the record the
+         * cursor is positioned on but there is a row larger than the searched-for key. This means
```

Comment:
The work row is a bit confusing. May be use record?

Reply:
I reworded in 99b0e3e6675b30bf5aaf257894825162904b0d6e, what do you think?